### ISSUE TITLE
Channels and Subscriptions down.

### ISSUE THREAD

**Issue by huhwhatwho** [yuliskov/SmartTube#3124]
This morning when I launched SmartTube, Channels and Subscriptions sections are not working and it's asking to login again.  Home, History and Search are working fine.

**Comment by trilias** [comment]
Yeah I have the same problem I try also new login but it not help

**Comment by Skwirl45** [comment]
Happening on both Beta and Stable releases, tried reinstalling, no difference

**Comment by Jean-PierreJ** [comment]
Same here. Updated last night before bed, noticed this issue in the morning.

What’s odd is that the app showed me logged in (i.e. I can see my Channels and my avatar), but going to Subscriptions keeps insisting that I “sign-in”. Navigating away to another menu item and returning to Subscriptions didn’t seem to change this (i.e. It wasn’t a refresh issue).

I did this two or three times (it’s early and my brain’s foggy, sorry), and each time I successfully login. It would then show me my Subscriptions but, as soon as I navigate away to another part of the app and return, it would disappear and I’d have to do this all over again.

As of right now, after signing in again the second/third time, it comes up fine when I go to Subscriptions.

Force-closing the app and re-opening also seems to work fine.

Fully rebooting/“restarting” the TV, however, and this happened again, showing me the “Can’t load content … Sign In” page; So back to the same … only if I sit on that page for about 20-30 seconds without (a) navigating away, (b) Signing in, or (c) pressing any buttons on my remote the page eventually does load. Once loaded, navigating away and returning works fine again. 

Following the same Reboot/“Restart” process on the TV again, this time without force closing the app first, opening the app and navigating to “Subscriptions”seems to bring the Subscriptions back again quickly/quicker and doesn’t show the Sign-In screen present.

Force closing the app, however, and rebooting/“restarting” the TV again (so that nothing is “persistent”) presents the same situation and this seems to be a consistent and repeatable bug for me but ONLY either after initial update OR if the app is force closed BEFORE rebooting the device.

To recap:

1. After updating the app appears to show the user avatar, as well as subscriptions under the Channel menu item, but NO subscription videos under the Subscriptions menu item; It only shows the Sign-In screen and this is a persistent issue.
2. After Signing In again (despite appearing or being signed in), it will still not load this Subscription information for roughly 20-30 seconds (I suspect the time may depend on the internet connection available, but cannot test that).
3. Rebooting the device WITHOUT force closing the app first, and then opening the app appears to still bring up the Subscription data under the Subscription menu item.
4. Force closing the app BEFORE rebooting the device, and then reopening the app after the reboot process, appears to show the Sign In splash screen on the Subscriptions menu item for approx. 20-30 seconds before loading the Subscriptions fully.
5. Once loaded, the Subscriptions seem to work fine again (a little slow to load, but only a few seconds longer than “normal”), and appears to work consistently until Step 4 is repeated.

App: SmartTube 20.36
Device: Sony XR-65X90K (Model BRAVIA 4K VH21)
OS: Android TV OS 10, Kernel version 4.19.75

**Comment by huhwhatwho** [comment]
It is working again.   Maybe it was a problem on the YouTube side.

**Comment by Munsterbuster** [comment]
Is it again a problem? I have issues since a couple of days with adding a subscription on both beta and stable. When I use YouTube manually, adding is no problem and they show immediately in SmartTube channels.
